apps/notifications/package.json

```diff
@@ -26,6 +26,7 @@
 	"dependencies": {
 		"@automattic/calypso-color-schemes": "workspace:^",
 		"@automattic/calypso-polyfills": "workspace:^",
+		"@automattic/i18n-utils": "workspace:^",
```

Comment:
If you want to test this:

1. cd `apps/notifications`.
2. run `yarn dev --sync`.
3. sandbox widgets.wp.com.
4. go to http://widgets.wp.com/notifications/